nav_normal.php

```diff
@@ -0,0 +1,13 @@
+<!--nav for User page -->
```

Comment:
26/8/2021: I also create a normal nav page so when I called it up the nav for normal user can be function. Also the link has been check so it all link to right location. The navigation bar hold the log out link, home page, Muiscplay page(all song sort by title (A to Z)) and when hover on it will display two column. One is link that sort by title and artist Z to A and a link to a page with a table of data sort the song by genre and artist and sort from (A to Z)